src/main/java/uk/gov/gchq/palisade/service/request/DataRequestConfig.java

```diff
@@ -16,16 +16,16 @@
 
 package uk.gov.gchq.palisade.service.request;
 
-import org.apache.commons.lang3.builder.EqualsBuilder;
-import org.apache.commons.lang3.builder.HashCodeBuilder;
 
 import uk.gov.gchq.palisade.Context;
+import uk.gov.gchq.palisade.Generated;
 import uk.gov.gchq.palisade.ToStringBuilder;
 import uk.gov.gchq.palisade.User;
 import uk.gov.gchq.palisade.resource.LeafResource;
 import uk.gov.gchq.palisade.rule.Rules;
 
 import java.util.Map;
+import java.util.Objects;
 
 import static java.util.Objects.requireNonNull;
 
```

Comment:
lines 48 needs a setter
setUser needs the generated tag and a null check 
getUser needs generated and doesnt need to check nulls

same for context and rules

Reply:
mutators also need updating